### ISSUE TITLE
NPM Audit fails - jsonwebtoken <= 8.5.1 has HIGH level security vulnerabilities

### ISSUE THREAD

**Issue by RobvH** [AzureAD/microsoft-authentication-library-for-js#5534]
### Core Library

MSAL Node (@azure/msal-node)

### Core Library Version

ALL

### Wrapper Library

Not Applicable

### Wrapper Library Version

NA

### Public or Confidential Client?

Public

### Description

Builds are failing with with all variants of MSAL that depend on jsonwebtoken <= 8.5.1 (thus, all right now?) fail on `npm audit` in Node environments and from end dependency scanners in browser environments. Migration to jsonwebtoken 9 is necessary and critical. 

Since `@azure/msal-node` and friends are depended on by many other packages (esp `@azure/identity` and thus ms-graph sdk etc), then this has widespread impact. 

### Error Message

```
# npm audit report

jsonwebtoken  <=8.5.1
Severity: high
jsonwebtoken unrestricted key type could lead to legacy keys usage  - https://github.com/advisories/GHSA-8cf7-32gw-wr33
jsonwebtoken has insecure input validation in jwt.verify function - https://github.com/advisories/GHSA-27h2-hvpr-p74q
jsonwebtoken's insecure implementation of key retrieval function could lead to Forgeable Public/Private Tokens from RSA to HMAC - https://github.com/advisories/GHSA-hjrf-2m68-5959
jsonwebtoken vulnerable to signature validation bypass due to insecure default algorithm in jwt.verify() - https://github.com/advisories/GHSA-qwph-4952-7xr6
fix available via `npm audit fix --force`
Will install @azure/identity@1.1.0, which is a breaking change
node_modules/jsonwebtoken
  @azure/msal-node  *
  Depends on vulnerable versions of jsonwebtoken
  node_modules/@azure/msal-node
    @azure/identity  >=1.2.0-alpha.20200903.1
    Depends on vulnerable versions of @azure/msal-node
    node_modules/@azure/identity

3 vulnerabilities (2 moderate, 1 high)
```

### Msal Logs

NA

### MSAL Configuration

```javascript
NA
```

### Relevant Code Snippets

```javascript
NA
```

### Reproduction Steps

`npm audit`

### Expected Behavior

`npm audit` doesn't report this as vulnerable.

### Identity Provider

Azure AD / MSA

### Browsers Affected (Select all that apply)

None (Server)

### Source

External (Customer)

**Comment by bmahall** [comment]
@RobvH This was addressed by [#5503](https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/5503/files) and should be part of next release (this coming week). Thanks!

**Comment by tnorling** [comment]
This has been released as msal-node 1.14.6
